### PR TITLE
OHRI-998 lab date format changed to date-time format

### PR DESCRIPTION
**Pull request by ODORA0**

### CODE REVIEW

**Comment by samuelmale** [comment]
@ODORA0 did you see the conflicts?

File: package.json [hunk 1]
```diff
@@ -70,32 +70,35 @@
...
+    "husky": "^3.0.4",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^24.9.0",
+    "jest-cli": "^24.9.0",
+    "openmrs": "^3.2.1-pre.1055",
+    "prettier": "^1.18.2",
+    "pretty-quick": "^1.11.1",
+    "webpack": "^5.37.0",
+    "webpack-cli": "^4.7.0",
+    "webpack-dev-server": "^4.2.1"
```
Comment (samuelmale) [inline package.json]: @ODORA0 did you touch this file by mistake? Do we need these changes?

File: src/components/inputs/date/ohri-date.component.tsx [hunk 2]
```diff
@@ -38,8 +39,26 @@ const OHRIDate: React.FC<OHRIFormFieldProps> = ({ question, onChange, handler })
     const refinedDate = date instanceof Date ? new Date(date.getTime() - date.getTimezoneOffset() * 60000) : date;
     setFieldValue(question.id, refinedDate);
     onChange(question.id, refinedDate, setErrors);
+    onTimeChange(false, true);
     question.value = handler.handleFieldSubmission(question, refinedDate, encounterContext);
   };
+
+  const onTimeChange = (event = false, useValue = false) => {
+    if (useValue) {
+      let prevValue = handler.getPreviousValue(question, encounterContext?.previousEncounter, fields);
```
Comment (samuelmale) [inline src/components/inputs/date/ohri-date.component.tsx]:
```js
const prevValue = handler.getPreviousValue(question, encounterContext?.previousEncounter, fields);
```

File: src/components/inputs/date/ohri-date.component.tsx [hunk 3]
```diff
@@ -38,8 +39,26 @@ const OHRIDate: React.FC<OHRIFormFieldProps> = ({ question, onChange, handler })
...
+      setTime(moment(prevValue.value).format('hh:mm'));
+    } else {
+      const time = event?.target?.value;
+      let currentDateTime = new Date(question.value.value);
```
Comment (samuelmale) [inline src/components/inputs/date/ohri-date.component.tsx]:
```js
 const currentDateTime = new Date(question.value.value);
```

**Review by samuelmale — CHANGES_REQUESTED** [review]
Just a few minor comments

**Comment by ODORA0** [comment]
Duplicate of https://github.com/UCSF-IGHS/openmrs-ohri-form-engine-lib/pull/43

**Pull request closed without merging**